README.md

```diff
@@ -33,5 +33,6 @@ The Community Committee is an autonomous committee that collaborates alongside t
 - Myles Borins ([MylesBorins](https://github.com/MylesBorins))
 - Ashley Williams ([ashleygwilliams](https://github.com/ashleygwilliams))
 - Gregor Martynus ([gr2m](https://github.com/gr2m))
+- Emily Rose ([emilyrose](https://github.com/emilyrose))
```

Comment:
I think I did local maximum alphabetical or something, good call on fixing it for the global maximum :)